test/testAnalysisTools.jl

```diff
@@ -0,0 +1,31 @@
+# Test for tree-based analysis tools found in `AnalysisTools.jl`.
+using Test
+using IncrementalInference
+using RoME # Used for loading canonical graphs (is this ok?).
```

Comment:
unfortunately not -- we don't really do circular dependencies for testing (although there is a special conditional snowflake in DFG tests).  Could we not add the required canonical factor graph to IIF instead -- do you need the manifolds for this test?